src/attendees/attendee.py

```diff
@@ -48,6 +56,8 @@ def get_attribute(self, key: Hashable) -> Any:
             TypeError if key is not hashable.
             KeyError if attendee does not have attribute with name key.
```

Comment:
Put colons after `TypeError` and `KeyError` so that the Sphinx docs generate properly